packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -359,8 +359,8 @@ class AppDomain extends Domain {
       connectionInfoCompleter = new Completer<DebugConnectionInfo>();
       connectionInfoCompleter.future.then((DebugConnectionInfo info) {
         Map<String, dynamic> params = <String, dynamic>{
-          'port': info.port,
-          'wsUri': info.wsUri
+          'port': info.httpUri.port,
```

Comment:
We need to send the port for a while for compatibility with old clients; sending both gives a grace period where old can work with new and vice versa.